<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
  <class>MainWindow</class>
  <widget class="QMainWindow" name="MainWindow">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>600</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>MainWindow</string>
    </property>
    <widget class="QWidget" name="centralWidget">
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,2">
        <item>
          <widget class="QGraphicsView" name="graphicsView"/>
        </item>
        <item>
          <layout class="QVBoxLayout" name="verticalLayout">
            <item>
              <widget class="QPlainTextEdit" name="plainTextEdit"/>
            </item>
            <item>
              <widget class="QPushButton" name="reloadCanvasButton">
                <property name="text">
                  <string>Reload Canvas</string>
                </property>
              </widget>
            </item>
          </layout>
        </item>
      </layout>
    </widget>
    <widget class="QMenuBar" name="menuBar">
      <property name="geometry">
        <rect>
          <x>0</x>
          <y>0</y>
          <width>855</width>
          <height>26</height>
        </rect>
      </property>
    </widget>
    <widget class="QStatusBar" name="statusBar"/>
    <action name="actionNew_File">
      <property name="text">
        <string>New File</string>
      </property>
    </action>
    <action name="actionOpen_File">
      <property name="text">
        <string>Open File</string>
      </property>
    </action>
    <action name="actionSave">
      <property name="text">
        <string>Save</string>
      </property>
    </action>
    <action name="actionSave_as">
      <property name="text">
        <string>Save as...</string>
      </property>
    </action>
    <action name="actionSave_all">
      <property name="text">
        <string>Save all</string>
      </property>
    </action>
    <action name="actionClose">
      <property name="text">
        <string>Close</string>
      </property>
    </action>
    <action name="actionClose_all">
      <property name="text">
        <string>Close all</string>
      </property>
    </action>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
  <resources/>
  <connections/>
</ui>
